### ISSUE TITLE
Jumping cursor in pygame (but ts_test works)

### ISSUE THREAD

**Issue by claudio755** [9and3r/mopidy-touchscreen#13]
Hi folks, 
im running touchscreen on Arch Linux 4.0.7-v7+ on a Raspberry Pi 2 rev. B together with a Spotpear 3.5" display.  
Everything works fine except the cursor, it bounces around all the time as soon as the touchscreen is touched, even though the finger is not moved. The given positions seem not to correlate to the touched position in any way. 
- I tested the display before so it should not be hardware related
- Its not a real mopidy-touchscrren issues, other pygame (test) scripts are reproducing the same problem
- Strangly ts_test works perfectly after calibration
- tslib version 1.1-1, SDL version 1.2.15, PyGame version1.9.1
- Settings are all right I guess
  `sdl_fbdev = /dev/fb1`
  `sdl_mousdrv = TSLIB`
  `sdl_mousedev = /dev/input/event0`
  `sdl_audiodriver = none`
  `sdl_path_dsp = /dev/null`
- I read somewhere the problem might be the SDL version which has a bug!? 

Any other ideas? I'm thankful for every hint! 
Cheer
Claude

**Comment by claudio755** [comment]
In the meanwhile I found SDL version 1.2.15-10 its probably causing the issue. See here:
https://forums.adafruit.com/viewtopic.php?f=47&t=76169
I downgraded to SDL 1.2.15-5 as suggested but still having the same issues. Hmm...

**Comment by ghost** [comment]
From my experience when ts_test works fine, pygame also works fine so no idea what the problem can be.

**Comment by claudio755** [comment]
Ok I finally got the issue solved - the problem is SDL version > 1.2.15-5 also mentioned in the link above.

I first tried to downgrade to version 1.2.15-5 with "pacman -U" which did not help.  I ended up downloading version 1.2.14 from https://www.libsdl.org/release/ and building it from scratch. 

Don't forget to include IgnorePkg   = sdl in nano /etc/pacman.conf to prevent an update of the package in future.
